forge-game/src/main/java/forge/game/ability/effects/CountersMoveEffect.java

```diff
@@ -300,10 +303,6 @@ public void resolve(SpellAbility sa) {
 
                         removeCounter(sa, source, cur, chosenType, counterNum, countersToAdd);
                     } else {
-                        if (!cur.canReceiveCounters(cType)) {
-                            continue;
-                        }
```

Comment:
then you shouldn't remove this part?

Reply:
that check was redundant because `removeCounter` does the same too